vignettes/GOstatsHyperG.Rmd

```diff
@@ -0,0 +1,527 @@
+---
+title: "How To Use GOstats Testing Gene Lists for GO Term Association"
+author:
+  - name: S. Falcon and R. Gentleman
+  - name: Dennis Ndubi
+    affiliation: "Vignette translation from Sweave to R Markdown / HTML"
+date: March 15, 2022
+package: GOstats
+bibliography: GOstats.bib
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document
+---
+# Introduction
+
+The`GOstats`{.sans-serif} package has extensive facilities for testing
```

Comment:
This still isn't quite right.  Please refer to the documentation about referring to package using Biocpkg